tests/generation/test_configuration_utils.py

```diff
@@ -679,18 +679,10 @@ def setUpClass(cls):
         cls._token = TOKEN
         HfFolder.save_token(TOKEN)
 
-    @staticmethod
-    def _try_delete_repo(repo_id, token):
-        try:
-            # Reset repo
-            delete_repo(repo_id=repo_id, token=token)
-        except:  # noqa E722
-            pass
-
     def test_push_to_hub(self):
+        repo_id = f"{USER}/test-generation-config"
```

Comment:
We don't really need to keep this - we can just create a repo. name.

But I just try to keep as it is